### DISCUSSION TITLE
Dynamic value in layout given from page

### DISCUSSION THREAD

**Discussion by WpSlt** [nuxt/nuxt#16436] [Questions]
Hello!
I would like to set text and loading component in layout and give control of these two elemet to the page.. but seem that layout with slot do not receive emit changes from page.

Some help please, i'm getting crazy :)

cheers

**Comment by TheAlexLichter** [comment]
Can you show us what you tried?

**Reply by WpSlt** [reply]
Here a sandbox with the situation:
https://codesandbox.io/s/smoosh-wind-3z7492?file=/layouts/testlayout.vue

**Reply by WpSlt** [reply]
In sandbox you can see that writing in page input emit should update the layout but it doesn't
